mne/io/base.py

```diff
@@ -1923,6 +1923,31 @@ def describe(self, data_frame=False):
             msg += f"{cols['max'][i]:>8.2f}"
             print(msg)
 
+    @verbose
+    def set_durations(self, mapping, verbose=None):
```

Comment:
I am -1 on adding so many public functions to Raw due to annotations. People should use:

`raw.annotations.set_durations(...)`

Reply:
Agreed. I only added as it was suggested for the other PR. I will remove.